### DISCUSSION TITLE
Flood simlution didn't run.

### DISCUSSION THREAD

**Discussion by weiyilan** [SynxFlow/SynxFlow#5] [Q&A] [answered]
<img width="227" alt="微信图片_20231205142720" src="https://github.com/SynxFlow/SynxFlow/assets/43534645/e50f33e2-d3c8-443b-b261-893dc30b991b">
<img width="809" alt="微信图片_20231205142712" src="https://github.com/SynxFlow/SynxFlow/assets/43534645/02fe74e9-ca3b-4990-a56f-8ff224f91a1c">

**Comment by weiyilan** [comment]
from synxflow import flood is not successful.

**Comment by xiaxilin** [comment]
Hi, can you double check whether there is CUDA-enabled GPU on your computer?

**Reply by sigurlily93** [reply]
May I ask why GPU is necessary ?

**Comment by xiaxilin** [comment, marked as answer]
I close this issue because the problem was solved by upgrating the cuda version to 11.7, with which the conda environment was compiled.
